From 2638c9d02af78844b3b11243725f68aed8782fe8 Mon Sep 17 00:00:00 2001
From: Meng Li <Meng.Li@windriver.com>
Date: Sat, 12 Nov 2022 17:07:43 +0800
Subject: [PATCH] configs: aptiv_cvc_sousa: add HSE_SECBOOT config

add HSE_SECBOOT config used to enable secure boot feature.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 configs/aptiv_cvc_sousa_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configs/aptiv_cvc_sousa_defconfig b/configs/aptiv_cvc_sousa_defconfig
index bf94a46583..35b56e846a 100644
--- a/configs/aptiv_cvc_sousa_defconfig
+++ b/configs/aptiv_cvc_sousa_defconfig
@@ -77,4 +77,5 @@ CONFIG_CMD_SMC=y
 CONFIG_FAT_WRITE=y
 CONFIG_MENU=y
 CONFIG_CMD_BOOTMENU=y
-CONFIG_XEN_SUPPORT=y
\ No newline at end of file
+CONFIG_XEN_SUPPORT=y
+CONFIG_HSE_SECBOOT=y
-- 
2.36.1

